### DISCUSSION TITLE
PHP 8.2+ support

### DISCUSSION THREAD

**Discussion by pmgarman** [snicco/fortress#13] [Feature Requests]
> Unlike other security plugins that sacrifice security for broad compatibility, Fortress is built exclusively for PHP 7.4/8.0/8.1, unlocking cutting-edge cryptography and security functionality.

8.2 & 8.3 are not listed in the EXCLUSIVE list of PHP versions Fortress is built for. Requesting a version of Fortress EXCLUSIVELY for 8.3 :)

**Comment by pmgarman** [comment]
![CleanShot 2024-02-16 at 16 02 40@2x](https://github.com/snicco/fortress/assets/502720/5c7e4037-6330-4b62-b4ba-e2f2574af331)

**Comment by calvinalkan** [comment]
That was probably a languaging error, you should have no issues on either of 8.2/8.3. 

CI is only running against 7.4-8.1 right now. 

Fortress's official v1 will most likely be raised to PHP8+ with a version policy the same as PHP core.
